packages/mobile/src/components/premium-track-purchase-drawer/PurchaseSummaryTable.tsx

```diff
@@ -9,31 +9,58 @@ const messages = {
   summary: 'Total',
   payExtra: 'Pay Extra',
   premiumTrack: 'Premium Track',
+  downloadableFiles: 'Downloadable Files',
   existingBalance: 'Existing Balance',
   total: 'Total',
   youPaid: 'You Paid',
   price: (price: string) => `$${price}`,
-  subtractPrice: (price: string) => `-$${price}`
+  subtractPrice: (price: string) => `-$${price}`,
+  included: 'included'
 }
 
 type PurchaseSummaryTableProps = {
   totalPriceInCents: number
   basePrice: number
   extraAmount?: number
+  // How many "streams" are available for purchase
+  // Prior to albums/bundles this should be 0 or 1.
+  streamPurchaseCount?: number
+  // How many "downloads" are available for purchase
+  // Prior to albums/bundles this should be 0 or 1.
+  downloadPurchaseCount?: number
+  // How many stems are available for purchase
+  stemsPurchaseCount?: number
 }
 
 export const PurchaseSummaryTable = ({
   totalPriceInCents,
   basePrice,
-  extraAmount
+  extraAmount,
+  streamPurchaseCount,
+  downloadPurchaseCount,
+  stemsPurchaseCount
 }: PurchaseSummaryTableProps) => {
-  const items: SummaryTableItem[] = [
-    {
+  const items: SummaryTableItem[] = []
+  if (streamPurchaseCount) {
+    items.push({
       id: 'premiumTrack',
       label: messages.premiumTrack,
       value: messages.price(formatPrice(basePrice))
-    }
-  ]
+    })
+  }
+
+  const downloadCount = (stemsPurchaseCount || 0) + (downloadPurchaseCount || 0)
+  if (downloadCount > 0) {
+    items.push({
+      id: 'premiumTrackDownload',
+      label: `${messages.downloadableFiles} (${downloadCount})`,
+      value: streamPurchaseCount
+        ? messages.included
+        : messages.price(formatPrice(basePrice)),
+      color: streamPurchaseCount ? 'subdued' : 'default'
```

Comment:
nit: this seems brittle - we're trying to alternate colors in the table rows right? there's gotta be a better way to do this, but 🤷

Reply:
not exactly. just show the color as lighter gray when the track is "stream for sale"